corgidrp/l1_to_l2a.py

```diff
@@ -49,24 +51,39 @@ def prescan_biassub(input_dataset, noise_maps=None, return_full_frame=False, det
                 raise Exception(f"Observation type of frame {i} is not 'SCI' or 'ENG' or 'ENG_EM' or 'EMG_CONV'")
 
         # Get the reliable prescan area
-        prescan = slice_section(frame_data, obstype, 'prescan_reliable', detector_regions)
+        prescan = slice_section(frame_data, obstype, 'prescan', detector_regions=detector_regions)
```

Comment:
I think this original version of this line should remain.  The argument `detector_regions` of `prescan_biassub` already handles `detector_regions`.

Reply:
Oh, never mind on this particular comment.  That's valid if we go this route where the reliable prescan can be accessed 2 different ways.